test/jackdaw/word_count_test.clj

```diff
@@ -0,0 +1,160 @@
+(ns jackdaw.word-count-test
```

Comment:
Can we move this over to the [example](https://github.com/FundingCircle/jackdaw/blob/master/examples/word-count/test/word_count_test.clj)? I think that might be a little more DRY, and the example could use a little TM love.

Reply:
I started with that but wasn't sure what the examples part of repo was trying to achieve. Is it an interactive demo/tutorial? Or a runnable test. It seems hard to keep it simple when it tries to be both. I think it's useful to have one complete e2e app and test as part of the test-suite.